package.json

```diff
@@ -58,16 +59,18 @@
     "eslint-plugin-markdown": "^2.2.1",
     "eslint-plugin-react": "^7.28.0",
     "eslint-plugin-react-hooks": "^4.3.0",
+    "glob": "^11.0.3",
     "jest": "^29.7.0",
     "style-loader": "^4.0.0",
     "terser-webpack-plugin": "^5.3.1",
+    "typescript": "^5.8.3",
     "uglify-js": "^3.15.5",
     "webpack": "^5.66.0",
     "webpack-cli": "^6.0.1",
     "webpack-dev-server": "^5.2.0"
   },
   "scripts": {
-    "build": "webpack --mode=production",
+    "build": "webpack --mode=production && node scripts/processDocoffFunctionDoc.js",
```

Comment:
This should not be needed as the ransformation ought to be done by the Webpack plugin.